docs/source/glossary.rst

```diff
@@ -64,11 +64,8 @@ Glossary
 
       The contract alias **must** use *one of* the following naming schemes:
 
-      -  ``<source-id>:<contract-name>``
-      -  ``<source-id>:<contract-name>[<identifier>]``
-
-      The ``<source-id>`` portion **must** match a key from the ``sources``
-      object for the respective source file of this contract type.
+      -  ``<contract-name>``
+      -  ``<contract-name>[<identifier>]``
```

Comment:
The idea to prefix a contract alias with `<source-id>` was reversed. Since source id's have no restrictions on characters, using a `:` to parse contract aliases (particularly when identifying a contract type of a dependency) is very difficult to handle.